test/tests-rv64-qemu-supervisor/app.toml

```diff
@@ -0,0 +1,52 @@
+name = "tests-rv64-qemu-supervisor"
+target = "riscv64imac-unknown-none-elf"
+board = "rv64-qemu-virt"
+stacksize = 2048
+chip = "../../chips/rv64-qemu-virt"
+
+[kernel]
+name = "demo-rv64-qemu-supervisor"
+requires = {flash = 28672, ram = 65536}
+features = []
+
+[tasks.runner]
```

Comment:
Does the test suite pass? It seems like it should hang on the timer check.

Reply:
It currently has no expectation of passing, for the issue you mentioned. I would only consider adding it to CI after we've sorted out the boot flow (which is currently bespoke)

Reply:
Okay, do you have instructions on how to get this running on top of an sbi in qemu?

Reply:
I think we should start putting together a care and feeding doc for this firmware component.